tools/gn

```diff
@@ -788,15 +788,9 @@ def to_gn_args(args):
   if args.enable_impeller_trace_canvas:
     gn_args['impeller_trace_canvas'] = True
 
-  if args.enable_impeller_vulkan:
-    gn_args['impeller_enable_vulkan'] = True
-
   if args.enable_impeller_vulkan_playgrounds:
     gn_args['impeller_enable_vulkan_playgrounds'] = True
```

Comment:
Remove `impeller_enable_vulkan_playgrounds` from `tools/gn` now that it no longer exists in the GN scripts